### PR TITLE
update sanity check for OpenFOAM 11

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

In https://github.com/OpenFOAM/OpenFOAM-dev/commit/968e60148ab31ec017f275673496d6193713d7e5 OpenFOAM combined some solvers.

### CODE REVIEW

**Review by ocaisa — APPROVED** [review]
Code looks good

**Comment by ocaisa** [comment]
Test report by @ocaisa
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _OpenFOAM-9-foss-2021a.eb_ 
 * **SUCCESS** _OpenFOAM-10-foss-2022a.eb_ 

Build succeeded for 2 out of 2 (2 easyconfigs in total)
login1 - Linux Rocky Linux 8.5, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz, Python 3.6.8
See https://gist.github.com/ocaisa/caa8825fc9dca08d75eba42e7a3c8903 for a full test report.

**Comment by ocaisa** [comment]
Test for OpenFoam 11 is in https://github.com/easybuilders/easybuild-easyconfigs/pull/18475

File: easybuild/easyblocks/o/openfoam.py [hunk 1]
```diff
@@ -438,14 +438,14 @@ def sanity_check_step(self):
...
             tools.remove("buoyantSimpleFoam")
-            tools.append("buoyantFoam")
-        # engineFoam replaced by reactingFoam in versions 10+
-        if self.is_dot_org and self.looseversion >= LooseVersion("10"):
             tools.remove("engineFoam")
-            tools.append("reactingFoam")
+            # both removed in version 11
+            if self.looseversion < LooseVersion("11"):
+                tools.append("buoyantFoam")
+                tools.append("reactingFoam")
```
Comment (boegel) [inline easybuild/easyblocks/o/openfoam.py:448]: out of curiosity: isn't there another (replacement) solver we can check for instead with OpenFOAM 11+?

**Pull request merged**